lib/inspec/resources/lxc.rb

```diff
@@ -47,11 +102,12 @@ def find_lxc_or_error
       raise Inspec::Exceptions::ResourceFailed, "Could not find `lxc`"
     end
 
-    def lxc_info_cmd
-      bin = find_lxc_or_error
-      info_cmd = "info #{@container_name} | grep -i Status"
-      lxc_cmd = format("%s %s", bin, info_cmd).strip
-      inspec.command(lxc_cmd)
+    def parse_command_output(output)
+      require "yaml" unless defined?(YAML)
+      YAML.load(output)
+    rescue JSON::ParserError => _e
```

Comment:
Why it will get JSON::ParseError?

Reply:
It will definitely not get JSON::ParseError. That's a mistake which I didn't notice.